zerver/lib/video_calls.py

```diff
@@ -24,3 +33,25 @@ def request_zoom_video_call_url(user_id: str, api_key: str, api_secret: str) ->
         return None
 
     return response.json()
+
+def get_redirect_uri(realm: Realm) -> str:
+    return realm.uri + "/json/calls/complete_zoom_user"
```

Comment:
It's possible that to support setups like zulipchat.com where each organization has their own subdomain, we'll need to do something more clever here that involves Zoom redirecting to `EXTERNAL_HOST/json/calls/complete_zoom_user?subdomain=%(realm.host)s`, not `realm.uri`, and us adding some extra code to deal with this going to back to the root domain. 

You can play with it after using the "Create realm" button; the key problem is we can't add every subdomain to Zoom's list of allowed redirect domains.